### ISSUE TITLE
runtime

### ISSUE THREAD

**Issue by madurner** [Cyanogenoid/dspn#7]
Hey! Nice work! I was wondering if you have any performance measures such as the runtime? E.g. how long does one iteration/sample take for the object detection task? thx

**Comment by Cyanogenoid** [comment]
Thanks! I'm using the default scripts with the associated hyperparams, a GTX 1080, and PyTorch 1.4. In bounding box prediction, on my system it takes on average 5 min 22 s for an epoch of training, which is 2187 batches of size 32. That is about 4.6 ms per sample, each sample being 10 iterations of the algorithm. For reference, these are about twice the time compared to the baseline MLP output.

Note that these include the gradient calculations and evaluation time from computing the Hungarian algorithm too. I haven't done much tuning of the batch sizes, so a bigger batch size might also be faster.

Let me know if there are any other numbers you want to know and I'll have a look.
